src/email_utils.py

```diff
@@ -23,6 +25,15 @@ def _attachment_inline(b64data, mime_type, filename, content_id):
 def send_email(logger: Logger, subject: str, msg: str, recipient: str | None = None, attachment: str | None = None) -> bool:
     if not recipient:
         recipient = app.config["ADMIN_EMAIL"]
+    if app.config["USE_SENDGRID"] is True:
+        #log_msg = f"Sending email via sendgrid"
+        #logger.info(log_msg)
+        return send_email_sendgrid(logger, subject, msg, recipient, attachment)
+    #log_msg = f"Sending email via postfix"
+    #logger.info(log_msg)
+    return send_email_postfix(logger, subject, msg, recipient, attachment)
+
+def send_email_sendgrid(logger: Logger, subject: str, msg: str, recipient: str | None = None, attachment: str | None = None) -> bool:
```

Comment:
```suggestion
    if app.config["USE_SENDGRID"]:
        return send_email_sendgrid(logger, subject, msg, recipient, attachment)
    return send_email_postfix(logger, subject, msg, recipient, attachment)

def send_email_sendgrid(logger: Logger, subject: str, msg: str, recipient: str, attachment: str | None = None) -> bool:
```

Reply:
it appears to be working... when i comment the SENDGRID_API_KEY in the .env file it looks like its sending via send_mail_postfix. its not actually sending the email but very promising. also if i do use the SENDGRID_API_KEY, it would send the email(im using the work address for the recipient) and i get the email.


removed the commented lines in send_mail function.

Reply:
changed 
```
recipient: str | None = None
```
to 
```
recipient: str
```